openbb_platform/core/openbb_core/provider/standard_models/company_news.py

```diff
@@ -25,13 +28,39 @@ class CompanyNewsQueryParams(QueryParams):
     limit: Optional[NonNegativeInt] = Field(
         default=20, description=QUERY_DESCRIPTIONS.get("limit", "")
     )
+    date: Optional[dateType] = Field(
```

Comment:
Did you consider removing this parameter `date` since we now have `start_date` and `end_date` it feels duplicated